releasenotes/notes/48769.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 33949
+releaseNotes:
+  - |
+    **Added** --wait and --timeout option to 'istioctl waypoint apply' command
```

Comment:
nit: I think the release note can be updated.

Reply:
ah good catch, I removed the mention of --timeout